examples/jenkins/pipeline/jenkinstemplate.json

```diff
@@ -243,12 +244,13 @@
     {
       "name": "JENKINS_PASSWORD",
       "displayName": "Jenkins Password",
-      "description": "Password for the Jenkins user.",
+      "description": "Password for the Jenkins 'admin' user.",
       "generate": "expression",
-      "value": "password"
+      "from": "[a-zA-Z0-9]{16}",
+      "required": true
     }
   ],
   "labels": {
-    "template": "jenkins-ephemeral-template"
```

Comment:
ahh, yes, sorry I was describing the dc and looking at the wrong field. Works for me.